File: Problems/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int prev = nums[0];
        int index = 1;
        int count = 1;
        for(int i = 1; i < nums.length; i++) {
            int curr = nums[i];
            if(prev == curr) {
                continue;
            } else {
                nums[index] = curr;
                prev = curr;
                index++;
                count++;
            }
        }
        return count;
    }
}

File: Problems/remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        i = 0
        prev = nums[i]

        for e in nums[1:]:
            if prev == e:
                continue
            else:
                i+=1
                nums[i] = e
                prev = e
        
        return i+1
